src/main/resources/static/openapi.yaml

```diff
@@ -249,158 +77,64 @@ paths:
+        200:
+          description: successful operation
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/QuestionList'
+                $ref: '#/components/schemas/UserSettings'
 
-    post:
-      tags: [ Question ]
-      summary: Create question
-      operationId: createQuestion
+    put:
+      tags: [ Settings ]
+      summary: Updated user settings
+      description: This can only be done by the logged in user.
+      operationId: updateSettings
       requestBody:
-        description: The object that describes the question we want to create.
+        description: Updated user settings
         content:
-          application/json:
+          'application/json':
             schema:
-              $ref: '#/components/schemas/CreateQuestionRequest'
-      responses:
-        '404':
-          description: The object is not found.
-          content: {}
-        '201':
-          description: Created
-          content: {}
-
-  '/api/v1/quizzes/{quizId}/questions/{questionId}':
-    parameters:
-      - $ref: '#/components/parameters/quizId'
-      - $ref: '#/components/parameters/questionId'
-
-    get:
-      tags: [ Question ]
-      summary: Get Question
-      operationId: readQuestion
+              $ref: '#/components/schemas/ChangeSettingsRequest'
+        required: true
       responses:
-        '200':
-          description: Question details
+        200:
+          description: ok
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/QuestionResponse'
-        '404':
-          description: Question doesn't exist
-          content: {}
+                $ref: '#/components/schemas/UserSettings'
 
-    delete:
-      tags: [ Question ]
-      summary: Delete Question
-      operationId: deleteQuestion
-      responses:
-        '200':
-          description: Question is deleted successfully
-          content: {}
-        '404':
-          description: Question doesn't exist
-          content: {}
-
-    put:
-      tags: [ Question ]
-      summary: Update question
-      operationId: updateQuestion
-      requestBody:
-        description: The object that describes the question we want to create.
-        content:
-          application/json:
-            schema:
-              $ref: '#/components/schemas/CreateQuestionRequest'
+  '/api/v1/data':
+    get:
+      tags: [ Air Quality ]
+      summary: Get the current Air Quality
+      operationId: readData
       responses:
-        '404':
-          description: The object is not found.
-          content: {}
-        '200':
-          description: The updated questions.
+        200:
+          description: successful operation
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/QuestionResponse'
-
+                $ref: '#/components/schemas/UserSettings'
```

Comment:
instead of UserSetting define an empty object AirQualityData in component/schemas and reference that object here.

Reply:
We have added the change